sycl/doc/extensions/proposed/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -541,9 +541,10 @@ node add(const device& syclDevice, T cgf, const property_list& propList = {});
 |This function adds a command group function object to a graph. The function
 object statically contains a group of commands, of which a single command is
 executed at runtime. A function object can be a host task which is scheduled by
```

Comment:
```suggestion
executed at runtime. A command group function object may submit any command
as defined by the core SYCL specification.
```

Reply:
Could extended this slightly to include extensions as well, "as defined by the SYCL specification, and any SYCL extensions unless explicitly stated otherwise"